lib/web/scripts/node-join/install.sh

```diff
@@ -261,6 +264,10 @@ log_cleanup_message() {
         log_only "- unload and remove Teleport launchd config ${LAUNCHD_CONFIG_PATH}/${LAUNCHD_PLIST_FILE}"
         log_only "  - launchctl unload ${LAUNCHD_CONFIG_PATH}/${LAUNCHD_PLIST_FILE}"
         log_only "  - rm -f ${LAUNCHD_CONFIG_PATH}/${LAUNCHD_PLIST_FILE}"
+
+        log_only "- remove any teleport application bundles"
+        for APP_BUNDLE in ${TELEPORT_APP_BUNDLES_darwin}; do EXAMPLE_DELETE_COMMAND+="/Applications/${APP_BUNDLE} "; done
```

Comment:
```suggestion
        for APP_BUNDLE in "${TELEPORT_APP_BUNDLES_darwin[@]}"; do EXAMPLE_DELETE_COMMAND+="/Applications/${APP_BUNDLE} "; done
```

Does this (and elsewhere) need to be quoted and expanded?